src/main/java/com/microsoft/teams/service/messaging/GetProjectsMessageHandler.java

```diff
@@ -18,17 +18,23 @@
 import com.atlassian.jira.avatar.Avatar;
 import com.atlassian.jira.avatar.AvatarService;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
 import org.apache.log4j.Logger;
```

Comment:
Do we not need to update log4j due to vulnerabilities discovered recently?

Reply:
we are not referencing any specific version of log4j